apps/spruce/src/pages/jobLogs/Metadata.tsx

```diff
@@ -5,13 +5,22 @@ import {
   MetadataTitle,
 } from "components/MetadataCard";
 import { StyledLink } from "components/styles";
+import { getParsleyCompleteLogsURL } from "constants/externalResources";
 import { JobLogsMetadata } from "./types";
 
 export const Metadata: React.FC<{
   loading: boolean;
   metadata: JobLogsMetadata;
-}> = ({ loading, metadata }) => {
+  isLogkeeper: boolean;
+}> = ({ isLogkeeper, loading, metadata }) => {
   const { sendEvent } = useJobLogsAnalytics(metadata.isLogkeeper);
+  const { execution, groupID, taskId } = metadata;
+  let completeLogsURL = "";
+  if (!isLogkeeper) {
+    completeLogsURL = getParsleyCompleteLogsURL(taskId, execution, groupID);
+  } else if (metadata.allLogsURL) {
+    completeLogsURL = metadata.allLogsURL;
+  }
```

Comment:
This should happen in `getMetadata` so that there is one source of truth for `allLogsURL`.

https://github.com/evergreen-ci/ui/blob/bfcdd72ec3c019ed1897c3fadb7809c7bc4dc671/apps/spruce/src/pages/jobLogs/utils.ts#L14